WordPress/gutenberg
# Docs: Remove list of keyboard shortcuts from FAQ page

Closes #60627

## What?

This PR completely removes the shortcut list from the FAQ documentation page.

## Why?

The current shortcut list is manually maintained, so updates to this page may be overlooked and may become out of date in the future. In order to provide a complete and accurate list, it should be sufficient to include the means to access the shortcut modal.

## How?

Since the method to access the shortcut modal was already described, I simply deleted only the shortcut list.

## Testing Instructions

Check out the shortcuts section of the new FAQ page:

https://github.com/WordPress/gutenberg/blob/docs/remove-faq-shortcut-list/docs/getting-started/faq.md#are-there-keyboard-shortcuts-for-gutenberg
